include/seastar/http/reply.hh

```diff
@@ -110,6 +110,7 @@ struct reply {
      * The content to be sent in the reply.
      */
     sstring _content;
+    size_t content_length = 0; // valid when received via client connection
```

Comment:
nit: uint64_t is more accurate here. A file larger than memory can be sent on 32-bit machines, though those no longer exist in servers.